enterprise/api/utils.py

```diff
@@ -0,0 +1,19 @@
+"""
+Utility functions for the Enterprise API.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.conf import settings
+
+
+WORKER_USERNAMES = (
```

Comment:
I think it would make sense to call them "service usernames" as it seems to be the clearest of the three, especially since the users are not exclusively in the context of async workers. I changed the variable name.